api/emoji.go

```diff
@@ -122,6 +122,10 @@ func createEmoji(c *Context, w http.ResponseWriter, r *http.Request) {
 		c.Err = result.Err
 		return
 	} else {
+		message := model.NewWebSocketEvent(model.WEBSOCKET_EVENT_EMOJI_ADDED, "", "", "", nil)
+		message.Add("emoji", result.Data.(*model.Emoji).ToJson())
+
+		go app.Publish(message)
```

Comment:
Hi @jwilander ,
thanks for review, I'm new to golang so correct me if my idea is wrong:
I checked current code (git grep "Publish") and saw most of them used coroutine (go keyword) when calling Publish function, so I took a look at Publish and saw it broadcast to all webhub we maintained. I supposed the reason why we used go coroutine was when webhub number becomes very large and it would be time consuming to wait for broadcast to return. So current  code uses coroutine to make it async and I followed their usage.
But I can change to use simple function call of course.